modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4687,6 +4681,16 @@ type Query {
   # Metadata for `enum FilterType`
   filterTypeMeta: [FilterTypeMeta!]!
 
+  # ITC execution results
+  itc(
+    programId:     ProgramId!
+
+    observationId: ObservationId!
+
+    # Whether to use cached results (true) or ignore the cache and make a remote ITC call (false).
+    useCache: Boolean! = true
+  ): ItcSuccess
```

Comment:
This had to move to the root since executing an effect at a lower level is apparently problematic.